_includes/download_release.html

```diff
@@ -0,0 +1,8 @@
+{% capture CURRENT_RELEASE %}{% for subver in page.release %}{{subver}}{% unless forloop.last %}.{% endunless %}{% endfor %}{% endcapture %}
+{% capture PATH_PREFIX %}/bin/bitcoin-core-{{CURRENT_RELEASE}}{% endcapture %}
+{% capture FILE_PREFIX %}bitcoin-{{CURRENT_RELEASE}}{% endcapture %}
+{% assign magnet=page.optional_magnetlink %}
+<https://bitcoincore.org{{ PATH_PREFIX }}/>
```

Comment:
I haven't tested this locally, but I think this code will give you the `.torrent` for the most recent release (e.g. 0.19.0.1 even if you're looking at the 0.17.1 page).  It should give you the correct magnet link.

Reply:
It doesn't; it uses `page.release`. But I renamed the confusingly named `CURRENT_RELEASE` to `RELEASE`.